static/css/header.css

```
@@ -40,13 +41,11 @@
 
 .navbar .links {
     padding: 0px 10px;
-    width: 100%;
     width: -webkit-fill-available;
+    width: 100%;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)